sqlitedict.py

```diff
@@ -105,6 +105,21 @@ def decode(obj):
     return loads(bytes(obj))
 
 
+def get_tablenames(filename):
```

Comment:
probably better to remove this function & move all code to method, wdyt?

Reply:
you mean just a class method?
I can do that. just makes use case look unpractical. see above

Reply:
@transfluxus what's about static method?

Reply:
ok